### PR TITLE
v1.2.0

### PR DESCRIPTION
**Pull request by derekprior**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -5,6 +5,24 @@ changelog, see the [CHANGELOG] for each version via the version links.
...
+
+### Fixed
+- Materialized view indexes are now properly dumped to `db/schema.rb`. This was
+  an oversight in previous releases, meaning `rake db:schema:load` was missing
+  indexes.
+- Calling `update_view` for a materialized view now properly finds associated
+  indexes for automatic reapplication. An issue in the previous index query was
+  returning no indexes.
+- The generators now accept namespaced view definitions. For example: `rails
+  generate scenic:view my_app.users`.
```
Comment (calebthompson) [inline NEWS.md]: This is a new feature right? Minor bump

Comment (derekprior) [inline NEWS.md]: We did have an issue for it earlier. I could go either way.

Comment (derekprior) [inline NEWS.md]: We'll go minor.

Comment (calebthompson) [inline NEWS.md]: I don't really care either. Bigger numbers are nicer and it feels like a feature but :+1: either way.

**Comment by calebthompson** [comment]
5 :exclamation:

**Comment by derekprior** [comment]
SCENIC X

**Pull request merged**